src/actions/index.js

```diff
@@ -1,28 +1,23 @@
-import content from '../apis/content';
-import { FETCH_EVENTS, DELETE_EVENT, MODAL_OPEN, MODAL_CLOSE } from './actionTypes';
+import { MODAL_OPEN, MODAL_CLOSE, GET_PAGES } from './actionTypes';
+import axios from 'axios';
 
-//Event Actions Examples
-export const fetchEvents = () => async (dispatch) => {
-  const res = await content.get('/events');
+const apiRoot = 'http://dev.huoa.org/wp-json';
 
-  dispatch({ type: FETCH_EVENTS, payload: res.data });
-};
-
-export const deleteEvent = (id) => async (dispatch) => {
-  // const res = await content.delete(`/events/${id}`);
-
-  dispatch({ type: DELETE_EVENT, payload: id });
+export const getPages = () => async (dispatch) => {
```

Comment:
oh wait i just noticed this you dont need to pass dispatch you can just return the object